src/storage/src/hummock/sstable/mod.rs

```diff
@@ -58,7 +59,7 @@ use super::{HummockError, HummockResult};
 
 const DEFAULT_META_BUFFER_CAPACITY: usize = 4096;
 const MAGIC: u32 = 0x5785ab73;
-const VERSION: u32 = 1;
+const VERSION: u32 = 2;
```

Comment:
To make the changes introduced by this PR backward compactible, modifying the `VERSION` const is not enough and we need to do the following things:
1. Change `SstableMeta::decode` to use different implementations to decode bloom filter based on the version.
2. When checking bloom filter in `surely_not_have_hashvalue`, using different implementations to check bloom filter based on the meta version.

Reply:
Indeed, but this will bring some duplicated code, maybe we can update version and remove duplicate code later.

Reply:
This is unavoidable if we want to ensure backward compatibility unless we fully deprecate a released version.

Reply:
We can keep the BTreeMap structure for bloom filter in SstableMeta and simply put only one entry in the BTreeMap for version 1. Then we use if-else in the decode and surely_not_have_hashvalue implementation to decide how to populate and check the BTreeMap.